sponnet/pipeline.libsonnet

```diff
@@ -36,15 +37,15 @@
 
   local artifact = self.artifact,
   artifacts:: {
-    bitbucketFile():: artifact('bitbucket/file'),
-    dockerImage():: artifact('docker/image'),
-    embeddedBase64():: artifact('embedded/base64'),
-    gcsObject():: artifact('gcs/object'),
-    githubFile():: artifact('github/file'),
-    gitlabFile():: artifact('gitlab/file'),
-    httpFile():: artifact('http/file'),
+    bitbucketFile():: artifact('bitbucket/file', 'custom'),
+    dockerImage():: artifact('docker/image', 'docker'),
+    embeddedBase64():: artifact('embedded/base64', 'base64'),
+    gcsObject():: artifact('gcs/object', 'custom'),
+    githubFile():: artifact('github/file', 'custom'),
```

Comment:
I think for most of these (check the source here) you can be more specific about the kind: https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/pipeline/config/triggers/artifacts/github/github.artifact.ts#L12

Reply:
Thank you for your comment ! It looks like a good change.
Should I change the other 'custom' to key too ?

Reply:
Yes I would change all the kinds that have some sort of adapter here: https://github.com/spinnaker/deck/tree/master/app/scripts/modules/core/src/pipeline/config/triggers/artifacts

Reply:
I changed the kinds.
I didn't change the kind of `kubernetesObject`, because I couldn't find the key.
https://github.com/spinnaker/spinnaker/pull/3741/files#diff-98560c97d4b23f0cf12af197f63a07b2R48